sunkit_image/tests/conftest.py

```diff
@@ -11,6 +11,7 @@
 import sunpy.data.sample
 import sunpy.map
 from astropy.utils.data import get_pkg_data_filename
+from scipy.ndimage import gaussian_filter
```

Comment:
Do we need to add scipy to our test dependencies?
I forgot if we have a core dependency scipy already.